api/Document.json

```diff
@@ -350,6 +350,214 @@
           }
         }
       },
+      "animationcancel_event": {
+        "__compat": {
+          "description": "<code>animationcancel</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/animationcancel_event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "54"
+            },
+            "firefox_android": {
+              "version_added": "54"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "animationend_event": {
+        "__compat": {
+          "description": "<code>animationend</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/animationend_event",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
The closest I can get to giving you a version number is [a comment from a later CL](https://chromium.googlesource.com/chromium/src/+/c03c815f179e89a75cc32daed34e2888a494ea66): "The flag was turned on for stable a month ago in M43." This applies to this, `chrome`, and `webview_android`.

Reply:
Thanks @jpmedley . In my update I've assumed that "M43" means "43", and that this would apply to all animation events (except animationcancel), and for all three targets.

Reply:
M43 is an internal shorthand. I try not to use it externally.

I think it does apply to all, but I will double-check for you.

Reply:
I apologize. I got the information wrong yesterday. I've opened a PR against this branch with the corrections.